SETUP.md

```diff
@@ -35,10 +35,16 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
     </details>
 
 1. `rake install`
+  * If you needed `bundle exec` in front of the previous command, you will need it here too.
+  * This can take a LONG time. You can see if progress is being made by opening up a second shell and staring `mysql -u root`. Run the following command twice, with approximately a 5-10 second delay between
```

Comment:
```suggestion
  * This can take a LONG time. You can see if progress is being made by opening up a second shell and starting `mysql -u root`. Run the following command twice, with approximately a 5-10 second delay between
```